{"name": "test_get_user_by_username", "status": "failed", "statusDetails": {"message": "AssertionError: False is not true", "trace": "self = <api.pet_store.tests.test_user.TestUser testMethod=test_get_user_by_username>\n\n    def test_get_user_by_username(self):\n        \"\"\"\n        Test Case #10: Get user nay username.\n        Testing username get details after adding one + acceptance and status code.\n        Pre_conditions (adding a new user) is being managed in the setUp function.\n        \"\"\"\n        logging.info(\"10_______TEST (USER) BEGAN_______10\")\n        get_user_response = self._pet_store.get_user_by_username(self.user_details.username)\n        # ASSERT\n>       self.assertTrue(get_user_response.ok)\nE       AssertionError: False is not true\n\ntests\\test_user.py:100: AssertionError"}, "description": "\n        Test Case #10: Get user nay username.\n        Testing username get details after adding one + acceptance and status code.\n        Pre_conditions (adding a new user) is being managed in the setUp function.\n        ", "attachments": [{"name": "log", "source": "606420dd-d2d8-4c19-bcf2-4e55e100253b-attachment.txt", "type": "text/plain"}], "start": 1722019012346, "stop": 1722019013309, "uuid": "4cf44305-765d-4f6a-a074-b1aa0f915f44", "historyId": "96a116f89fcb0de8e40d1dd56a10f85d", "testCaseId": "96a116f89fcb0de8e40d1dd56a10f85d", "fullName": "tests.test_user.TestUser#test_get_user_by_username", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "DESKTOP-DB3BQ5K"}, {"name": "thread", "value": "10536-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}